src/modules/vmware/application/dto/vmware-connection.dto.ts

```
@@ -0,0 +1,34 @@
+export class VmwareConnectionDto {
+  @ApiProperty({
+    description: 'IP address or hostname of the vCenter/ESXi server',
+  })
+  @IsString()
+  host: string;
+
+  @ApiProperty({ description: 'Username for vCenter/ESXi authentication' })
+  @IsString()
+  user: string;
+
+  @ApiProperty({ 
```

ESLint

[error] Delete `·`

(prettier/prettier)